src/rp2_common/hardware_rtc/rtc.c

```diff
@@ -13,7 +13,14 @@
 
 // Set this when setting an alarm
 static rtc_callback_t _callback = NULL;
-static bool _alarm_repeats = false;
+
+typedef enum {
+    NO_REPEAT                =  0,
+    CONTINUOUS_REPEAT        =  1,
+    CONTINUOUS_REPEAT_ON_SEC =  2,
```

Comment:
This looks like it might be a typo for `CONTINUOUS_REPEAT_ONE_SEC` :wink: Perhaps `CONTINUOUS_REPEAT_EVERY_SECOND` might be less ambiguous?

Reply:
Not a typo, just a different point of view. An alarm is triggered ON change of x, where x may be SEC or MIN or HOUR, but all other possible alarms/triggers are already included in CONTINUOUS_REPEAT.
From SW pov you are right, I'll modify it